angular-intro-project/src/app/add-course/add-course.component.ts

```diff
@@ -13,32 +14,46 @@ export class AddCourseComponent implements OnInit {
   courseDate: number;
   courseDuration: number;
   id: number;
+  course: Course;
 
   constructor(private route: ActivatedRoute, private router: Router, private courses: CoursesService) { }
```

Comment:
If you inject services consider use their names with `Service` postfix. Like `coursesService`